libs/domain/order/heading/index.ts

```diff
@@ -0,0 +1 @@
+export * from './heading.component';
```

Comment:
same for styles and def

Reply:
there are no styles and we don't want to export def